main.py

```diff
@@ -4,8 +1,52 @@
+import json
+from db.models import Race, Skill, Guild, Player
+from typing import Any, Optional
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data: dict[str, Any] = json.load(file)
```

Comment:
The use of type hints like 'dict[str, Any]' is valid in Python 3.9+, but ensure your environment supports this syntax. If you're targeting earlier versions, consider using 'Dict[str, Any]' from the 'typing' module.